shared/src/business/utilities/pdfGenerator/documentTemplates/PrintableWorkingCopySessionList.tsx

```diff
@@ -25,6 +25,14 @@ export const PrintableWorkingCopySessionList = ({
       ? `${formattedTrialSession.formattedStartDateFull} - ${formattedTrialSession.formattedEstimatedEndDateFull}`
       : `${formattedTrialSession.formattedStartDateFull}`;
 
+  const trialStatusCounts = formattedCases.reduce((counters, c) => {
```

Comment:
This will be up to Gunnison to decide, but in general we have tried to keep logic outside of the components and inside, e.g., Cerebral computed or state. (The idea, although imperfect, was to separate logic out from rendering. You'll notice we don't really have react-specific testing in our unit tests; we rely on the separation between state and rendering, testing the state.) I don't think I have the authority to say, "Don't do this!" but I wanted to let you know.

Reply:
I agree with Chris! In keeping with the established patterns, we should compute this elsewhere and pass it in here.